Plugin.pm

```diff
@@ -1130,7 +1126,7 @@ sub QobuzGetTracks {
 			}
 
 			push @$items, {
-				name  => cstring($client, 'PLUGIN_QOBUZ_RELEASED_AT') . cstring($client, 'COLON') . ' ' . Slim::Utils::DateTime::shortDateF($album->{released_at}),
+				name  => cstring($client, 'PLUGIN_QOBUZ_RELEASED_AT') . cstring($client, 'COLON') . ' ' . $album->{release_date_stream},
```

Comment:
Is the `release_date_stream` string localized? Or unpredictable? Otherwise it would be nice if it could be localized for the user. I'm always confused by "2023-06-05"... is this in May or June?...

Reply:
I believe that it's always in YYYY-MM-DD format. It could easily be localized with a few additional instructions if that would be preferable.

Reply:
Yes, please. I can't get used to any other date format than "ours" 😀.

Reply:
Coming in the next PR.